utils.py

```diff
@@ -453,3 +454,30 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+cache_slug = None  # pylint: disable=invalid-name
+def get_cache_slug():
+    """Returns appropriate cache slug depending whether dev or prod. This cache
+    slug should only be used for urls in html, css and js files since it
+    contains a forward slash.
+    """
+    global cache_slug # pylint: disable=global-statement, invalid-name
+    if cache_slug is None:
```

Comment:
Ah, I was looking from a different angle. 
We will save 4 spaces :).
Either is fine for me, this was a just personal suggestion.